src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/LocalSlotMappingTests.cs

```diff
@@ -246,13 +246,13 @@ .maxstack  2
   </methods>
 </symbols>
 ");
-            var debugInfoProvider = v0.CreatePdbInfoProvider();
+            var symReader = v0.CreateSymReader();
```

Comment:
Then can we remote the .Dispose (assuming it doesn't change the observable behavior)?